### ISSUE TITLE
feat: `getChatsByLabel`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1503]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.9.12](https://github.com/open-wa/wa-automate-nodejs/compare/3.9.11...3.9.12) (2021-03-21)

- Release 3.9.12 [`33b1885`](https://github.com/open-wa/wa-automate-nodejs/commit/33b1885b56aa9aab74afda143bdf004a802841b7)
- ✨ feat: implemented `getChatsByLabel` #1503 [`49e37b2`](https://github.com/open-wa/wa-automate-nodejs/commit/49e37b25343657b105d1c0d000735359f5f19c82)
- 🥚 not a contact error patched [`33452cb`](https://github.com/open-wa/wa-automate-nodejs/commit/33452cb00d67176b25140970297e0c5fef6b3d56)
- 🥚 patch update [`2530868`](https://github.com/open-wa/wa-automate-nodejs/commit/25308687f4e5a5f800b31bac0f8fd6bdf723fa23)

**Comment by smashah** [comment]
https://docs.openwa.dev/classes/api_client.client.html#getchatsbylabel
